### PR TITLE
[fix](Nereids) fix sql cache return old value when truncate partition

### PR DESCRIPTION
**Pull request by 924060929**
1. fix sql cache return old value when truncate partition
2. use expire_sql_cache_in_fe_second to control the expire time of the sql cache which in the NereidsSqlCacheManager

### CODE REVIEW

**Comment by 924060929** [comment]
run buildall

**Comment by 924060929** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -1314,12 +1314,18 @@ public class Config extends ConfigBase {
      *  Minimum interval between last version when caching results,
      *  This parameter distinguishes between offline and real-time updates
      */
+    @ConfField(mutable = true, masterOnly = false)
+    public static int cache_last_version_interval_second = 30;
```
Comment (morrySnow) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:1318]: add description in annotation

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 2]
```diff
@@ -1314,12 +1314,18 @@ public class Config extends ConfigBase {
...
+    /**
+     *  Expire sql sql in frontend time
+     */
     @ConfField(
             mutable = true,
             masterOnly = false,
             callbackClassString = "org.apache.doris.common.NereidsSqlCacheManager$UpdateConfig"
     )
-    public static int cache_last_version_interval_second = 30;
+    public static int expire_sql_cache_in_fe_second = 300;
```
Comment (morrySnow) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:1328]: add description in annotation

**Review by morrySnow — APPROVED** [review]

**Pull request merged**